### PR TITLE
[#171207601] Create module cosmosdb_sql_container

### PR DESCRIPTION
**Pull request by uolter**

### CODE REVIEW

File: azurerm_cosmosdb_container/vars.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  description   = "The name of the environment"
+}
+
+variable "name" {
+  type        = string
+  description = "The name of the Cosmos DB instance."
+}
+
+variable "resource_group_name" {
+    type        = string
```
Comment (rchessa) [inline azurerm_cosmosdb_container/vars.tf]: leave just 3 spaces before type and align all the other variables

File: azurerm_cosmosdb_container/vars.tf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+variable "database_name" {
+    type        = string
+    description = "The name of the Cosmos DB SQL Database to create the container within."
+} 
+
+variable "partition_key_path" {
+    type        = string
+    description = "Define a partition key."
+    default     = null
+
```
Comment (rchessa) [inline azurerm_cosmosdb_container/vars.tf]: Remove blank line

File: azurerm_cosmosdb_container/vars.tf [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}              
+
+variable "unique_key_paths" {
+    type        = list(string)
+    description = "A list of paths to use for this unique key."         
+}
+
+locals {
+  resource_name         = "${var.global_prefix}-${var.environment}-cosmosdb-${var.name}"
+}
```
Comment (rchessa) [inline azurerm_cosmosdb_container/vars.tf]: Add newline at the end of the file

File: azurerm_cosmosdb_container/main.tf [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt
+  backend "azurerm" {}
+}
+
+resource "azurerm_cosmosdb_sql_container" "cosmosdb_sql_container" {
+  name                      = local.resource_name
+  resource_group_name       = var.resource_group_name
```
Comment (rchessa) [inline azurerm_cosmosdb_container/main.tf]: Align = with resource_group_name, and all the other variables

File: azurerm_cosmosdb_container/vars.tf [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  default     = null
+}
+
+variable "unique_key_paths" {
+  type        = list(string)
+  description = "A list of paths to use for this unique key."
+}
+
+locals {
+  resource_name = "${var.global_prefix}-${var.environment}-cosmosdb-${var.name}"
```
Comment (francescopersico) [inline azurerm_cosmosdb_container/vars.tf]: The scope of this resource is the Database so you don't need to create a complex name with prefix. Just use the var.name passed.

File: azurerm_cosmosdb_container/main.tf [hunk 6]
```diff
@@ -0,0 +1,25 @@
+provider "azurerm" {
```
Comment (francescopersico) [inline azurerm_cosmosdb_container/main.tf]: Please rename the directory to "azurerm_cosmosdb_sql_container"

**Pull request merged**